addons/web/models/base_document_layout.py

```diff
@@ -138,8 +138,10 @@ def _compute_preview(self):
                     'preview_css': preview_css,
                     'is_html_empty': is_html_empty,
                 })
+                return ir_ui_view, wizard_with_logo, preview_css, is_html_empty
             else:
                 wizard.preview = False
+                return False, False, False, False
```

Comment:
![image](https://github.com/odoo/odoo/assets/6582885/8abdc20e-1950-428d-9e8e-fa0a91123cc5)

We can't return values in a compute function.
Instead of doing that, we can create a helper function for `preview_css = markupsafe.Markup(self._get_css_for_preview(styles, wizard_with_logo.id))` (if you would like, I mean it's a oneline), and other values can be get either by importing the `is_html_empty` either by using the wizard record.